lib/BaseViewer.js

```diff
@@ -655,11 +655,9 @@ BaseViewer.prototype._createContainer = function(options) {
 
   var container = domify('<div class="bjs-container"></div>');
 
-  assign(container.style, {
-    width: ensureUnit(options.width),
-    height: ensureUnit(options.height),
-    position: options.position
-  });
+  container.style.width= ensureUnit(options.width);
```

Comment:
Use `applyStyles` here.

Reply:
I've made this change. In doing so I moved the `applyStyles` util into `lib/import/Util.js` to make it more logically accessible to other parts of the library.